### PR TITLE
`getCircleInnerDiameter()` on holed components

### PR DESCRIPTION
**Pull request by keckler**
## Description

Add a `getCircleInnerDiameter()` method to all Holed components. This is in response to #755 .

Also filled out the relevant component parameter definitions.

---

## Checklist

- [x] The code is understandable and maintainable to people beyond the author.
- [x] Tests have been added/updated to verify that the new or changed code works.
- [x] There is no commented out code in this PR.
- [x] The commit message follows [good practices](https://terrapower.github.io/armi/developer/tooling.html).
- [x] All docstrings are still up-to-date with these changes.

### CODE REVIEW

File: armi/reactor/components/componentParameters.py [hunk 1]
```diff
@@ -156,9 +156,9 @@ def getHoledHexagonParameterDefinitions():
 
     pDefs = parameters.ParameterDefinitionCollection()
     with pDefs.createBuilder(location=ParamLocation.AVERAGE, saveToDB=True) as pb:
-        pb.defParam("holeOD", units="?", description="?")
+        pb.defParam("holeOD", units="cm", description="Diameter of interior hole(s)")
 
-        pb.defParam("nHoles", units="?", description="?")
+        pb.defParam("nHoles", units="cm", description="Number of interior holes")
```
Comment (albeanth) [inline armi/reactor/components/componentParameters.py:161]: Solid addition.

**Comment by albeanth** [comment]
Looks good to me! :shipit: 🚀

File: armi/reactor/components/componentParameters.py (hunk 1 above)
Comment (jakehader) [reply armi/reactor/components/componentParameters.py:161]: Why does the number of holes have a unit of cm?

**Comment by john-science** [comment]
@keckler  `black` needs to be run.

(I would do it myself, but it's in your repo. Sorry.)

**Comment by keckler** [comment]
I'm not sure what's up with that. On my machine, `black` says everything is fine:
image

And looking at the changes, I don't see anything amiss from a formatting perspective.

Maybe re-trigger the tests?

**Comment by albeanth** [comment]
@keckler your feature branch is 1 commit behind armi::main. I wonder if that could be an issue?

**Comment by keckler** [comment]
Hm, my `click` version is behind. Let's try that.

**Comment by keckler** [comment]
Okay, it was `click`. 

I just pushed up a new reformatted version.

Sorry about all the hassle!

**Comment by keckler** [comment]
This should be good to merge now.

**Pull request merged**